externs/browser/html5.js

```diff
@@ -4286,3 +4319,60 @@ Plugin.prototype.length;
 
 /** @type {string} */
 Plugin.prototype.name;
+
+/**
+ * @see https://html.spec.whatwg.org/multipage/scripting.html#custom-elements
+ * @constructor
+ */
+function CustomElementRegistry() {}
+
+/**
+ * @param {string} tagName
+ * @param {!function(new:HTMLElement)} klass
+ * @param {{extends: string}=} options
+ */
+CustomElementRegistry.prototype.define = function (tagName, klass, options) {};
+
+/**
+ * @param {string} tagName
+ * @return {?function(new:HTMLElement)}
+ */
+CustomElementRegistry.prototype.get = function(tagName) {};
+
+/**
+ * @param {string} tagName
+ * @return {Promise<!function(new:HTMLElement)>}
+ */
+CustomElementRegistry.prototype.whenDefined = function(tagName) {};
+
+/** @type {!CustomElementRegistry} */
+var customElements;
+
+/**
+ * @constructor
+ * @extends {HTMLElement}
+ */
+function HTMLSlotElement() {}
+
+/**
+ * @param {!{flatten: boolean}=} options
+ * @return {!Array<!Node>}
+ */
+HTMLSlotElement.prototype.assignedNodes = function(options) {};
+
+/** @type {boolean} */
+Event.prototype.composed;
+
+/**
+ * @return {!Array<!(Element|ShadowRoot|Document|Window)>}
+ */
+Event.prototype.composedPath = function() {};
+
+/** @constructor */
+function InputDeviceCapabilities(){}
```

Comment:
InputDeviceCapabilities has another boolean property `pointerMovementScrolls`.

InputDeviceCapabilities can be constructed with an optional bag of `{firesTouchEvents: boolean|undefined, pointerMovementScrolls: boolean|undefined}`